src/containers/Review/overview/Overview.tsx

```diff
@@ -68,9 +87,57 @@ export const Overview: React.FC<{
                 <strong>{strings.REVIEW_OVERVIEW_SERIAL}: </strong>
                 {serial}
               </p>
+              {applicantDeadline && deadlineActive && (
+                <p className="right-item">
+                  <strong>{strings.REVIEW_OVERVIEW_DEADLINE}: </strong>
+                  {DateTime.fromJSDate(applicantDeadline).toLocaleString(DateTime.DATETIME_SHORT)}
+                </p>
+              )}
             </div>
+            {applicantDeadline &&
+              (outcome === ApplicationOutcome.Expired ||
+                current.status === ApplicationStatus.ChangesRequired) && (
```

Comment:
do we really need to be able to extend when `ChangesRequired` ? What would be the difference between it being `ChangesRequired` and `Draft` (i.e. when it's in LOQ and being edited by the user)

Reply:
Just trying to think through the condition here. It needs to be active when:
- a deadline exists(active or other)
- application is expired
- application is not expired but in Draft or ChangesRequested

But NOT active when:
- application is APPROVED or REJECTED or WITHDRAWN (i.e. it's properly "Done")

So, okay -- I think this covers it: 8d022747

Any case I've missed?

Reply:
Hmm, I think it should just be whenever application is in the hand of applicant, it's a bit tricky actually:

 status `draft` or `changes_required` 
 or
 status 'completed' and outcome 'expired'
 would be great as a method `is_applicant_action` (ts method), although would also be great as gql resolver on application

Reply:
It can't include the case when it's a first draft, as there's nothing to expire yet. Though there wouldn't be a deadline event created yet, so that's alright.

> status 'completed' and outcome 'expired'

The status is redundant -- it's always COMPLETED when outcome is anything other than "PENDING" (database enforced)

Okay, thinking about it a bit more clearly this morn, I think this should do it:  1565b252

Basically, must meet ALL of the following criteria:
- deadline exists
- status either DRAFT or CHANGES_REQUIRED
- outcome either PENDING or EXPIRED

Sound right?

Reply:
I thought that when outcome goes to anything other then pending, status auto changes to `completed`  ?

I still think logic should be as per my last comment

Reply:
Okay -- this time?: 6746cb7e 🤞

I think that's pretty much what I started with, except the extra "DRAFT" option.